db/schema.rb

```diff
@@ -126,6 +126,8 @@
     t.text "terms_and_conditions"
     t.integer "board_layout", default: 0, null: false
     t.boolean "registration_enabled", default: true, null: false
+    t.boolean "ask_for_location", default: false, null: false
+    t.boolean "location_required", default: false
```

Comment:
One of these cannot be set to null and the other can, not sure if this was done on purpose or just missed for location_required.